yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -77,6 +79,12 @@ func (yb *TargetYugabyteDB) Exec(query string) (int64, error) {
 
 	res, err := yb.db.Exec(query)
 	if err != nil {
+		var pgErr *pgconn5.PgError
+		if errors.As(err, &pgErr) {
+			if pgErr.Hint != "" || pgErr.Detail != "" {
+				return rowsAffected, fmt.Errorf("run query %q on target %q: %w \nHINT:%q\nDETAIL:%q", query, yb.tconf.Host, err, pgErr.Hint, pgErr.Detail)
```

Comment:
space
`\nHINT: %q\n`

Also do we need %q here, or %s is sufficient. In case the Hint/Detail contains tabs/newline they will be quoted.